_examples/slices/test.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 The go-python Authors.  All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from __future__ import print_function
+import slices
+
+a = [1,2,3,4,5,6]
```

Comment:
Ditto: please put the same content in the list than in the Go slice. Less confusing to debug.